docs/pages/try-out-teleport/local-kubernetes.mdx

```diff
@@ -83,8 +83,8 @@ Verify that Teleport is running.
 
 ```code
 $ kubectl get pods
-NAME                               READY   STATUS    RESTARTS   AGE
-teleport-cluster-b9dc6c68b-d75fb   1/1     Running   0          46s
+# NAME                               READY   STATUS    RESTARTS   AGE
```

Comment:
I don't like commenting out the output - I find it confusing.

The goal of a code snippet should either be:

1. Make it easy to copy a long or error-prone command
2. Demonstrate what the output of a typical command looks like

As you've seen here, it's hard to do both well.

In this case, I don't think we need to optimize for copying `kubectl get pods`. It's a command that kube users are very familiar with, and hard to get wrong.

If you really want to solve for both items above, we could do separate blocks. One for the command, and then another ```text block that shows what the output might look like.

Reply:
I've adjusted the use of comments for output in this PR.